modules/vba_documents/spec/models/upload_submission_spec.rb

```diff
@@ -557,6 +563,10 @@ def status_complete_packets(*reasons)
     it 'returns false when status is error and the error code is DOC2XX (upstream error)' do
       expect(upload_error_upstream.in_final_status?).to be(false)
     end
+
+    it 'returns true when status is error and the error code is DOC2XX (upstream error) and 14 days or older' do
+      expect(upload_error_upstream_old.in_final_status?).to be(true)
+    end
```

Comment:
Is there a way to make this spec auto-update if the value of the `MAX_UPSTREAM_ERROR_AGE_DAYS` constant changes in the future?

For example, you could have the assertion mention the constant, e.g. "...and older than MAX_UPSTREAM_ERROR_AGE_DAYS" instead of mentioning 14 days directly. And you could use the constant when creating the `upload_error_upstream_old` object, e.g. `created_at: (VBADocuments::UploadSubmission::MAX_UPSTREAM_ERROR_AGE_DAYS + 1).days.ago`.

Reply:
done